kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/exercises/homework/FlightFinder.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.collections.adv.exercises.homework;
+
+import java.util.*;
+import java.util.List;
+
+public class FlightFinder {
+    private List<Flight> flights = FlightRepository.getFlightsTable();
+
+    public FlightFinder(List<Flight> flights) {
+        this.flights = flights;
+    }
+
+    public FlightFinder() {
+        return;
+    }
+
+    public List<Flight> findFlightsFrom(String departure) {
+        List<Flight> flightFrom = new ArrayList<>();
+        for (Flight flight : FlightRepository.getFlightsTable()) {
```

Comment:
po kiego to wywolanie getFlightsTable() ? Przeciez masz to juz w tym obiekcie przekazane przez konstruktor i obecne w polu flights.